server/xpub-model/entities/user/helpers/elife-api.js

```diff
@@ -13,16 +13,20 @@ const request = (endpoint, query = {}) => {
   if (secret) {
     req.header.Authorization = secret
   }
-  return req
-    .query(query)
-    .catch(err => {
-      logger.error('Failed to fetch from eLife API:', err.message)
-      throw err
-    })
+  return req.query(query).catch(err => {
+    logger.error('Failed to fetch from eLife API:', err.message)
```

Comment:
can we log anything more than `err.message`, does the object have a stacktrace or line or file?

Reply:
The object does have a stacktrace, we should be able to log that.

Reply:
I have included `err.stack` when logging the error now